src/crawlers/crawler_all_artists.py

```diff
@@ -0,0 +1,52 @@
+import os
+import sys
+import pandas as pd
+from bs4 import BeautifulSoup
+sys.path.append(os.path.abspath('../config'))
+sys.path.append(os.path.abspath('../crawler_utils'))
+from scraping_utils import ScrapingUtils as su
+from discography_utils import DiscographyUtils as du
+from common_utils import GenericUtils as gu
+
+root_link = 'https://www.azlyrics.com'
+div_tag = 'div'
+a_tag = 'a'
+div_attr = {'class': 'col-sm-6'}
+
+
+class AllArtists:
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def scrape_all_artists():
+        artists = []
+
+        for index in range(1, 28):
+            # Scraping Mechanism
+            contents = gu.read_contents(f'../../webpage/all_artists/page{index}.html')
+            soup = BeautifulSoup(contents, 'lxml')
+
+            scraped_cols = su.bs4_scrape(soup, 'find_all', div_tag, div_attr)
+
+            for col in scraped_cols:
+                scraped_col = su.bs4_scrape(col, 'find_all', a_tag)
+
+                for r in scraped_col:
+                    artists.append(r.text)
```

Comment:
Since artists is a dictionary now, this would become:
```python
artists[r.get('href')] = r.text
```

Reply:
I've got a question. Let's say for the sake of flexibility, we want to store this dictionary in a Dataframe. How would I store this dictionary ('VA':'VB') inside the frame where 'Column1' -> VA1, VA2, .  . . , VAn  (Links), and 'Column 2' -> VB1, VB2, . . . , VBn  (Artists)?

Reply:
The above dictionary returned would be in this format:
```python
data = {"Artists": {"a/a1.html": "a1", "f/floyda1bentley.html": "A1", "a/a1xj1.html": "A1 x J1", "a/a.html": "A", "a/a2h.html": "A2H", "a/a36.html": "A36", "a/a4.html": "A4", "a/a92.html": "A92", "s/snohaalegra.html": "Snoh Aalegra", "a/aaliyah.html": "Aaliyah", "s/saaraaalto.html": "Saara Aalto", "a/aaradhna.html": "Aaradhna", "a/aarne.html": "Aarne", "a/aaroncarpenter.html": "Carpenter, Aaron", "c/carter.html": "Carter, Aaron", "a/aaroncole.html": "Cole, Aaron", "a/aarondoh.html": "Doh, Aaron"}
    }
```

Here's just a sample. You only would have to do this: `pd.DataFrame(data)`

Reply:
Right. The df needs to be Reset first after Conversion. Then, the Columns need to be initialized.
Initially I tried this:
```py
artists_df = pd.DataFrame(data, columns=["Artist URL", "Artist Name"]).reset_index()
```
But, instead got an Empty df.